examples/with-mtls/README.md

```
@@ -0,0 +1,633 @@
+}
+
+// 3. Configure Auth0Client with mTLS
+export const auth0 = new Auth0Client({
+  useMtls: true,           // Use TlsClientAuth(), no client_secret
+  customFetch: mtlsFetch,  // All requests use client certificate
+});
+```
+
+### Authentication Flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)